modules/cf-core/src/methods/state-channel/create/controller.ts

```diff
@@ -65,12 +65,15 @@ export default class CreateChannelController extends NodeController {
       initiatorXpub: publicIdentifier
     });
 
+    // use state channel for owners
+    const sc = await store.getStateChannel(multisigAddress);
+
     const msg: CreateChannelMessage = {
       from: publicIdentifier,
       type: NODE_EVENTS.CREATE_CHANNEL,
       data: {
         multisigAddress,
-        owners,
+        owners: sc.multisigOwners,
```

Comment:
Why not use the owners that are passed in? Since this is the create channel method they should never be different and would avoid a DB read.

Reply:
It comes out to xpubs vs eth addresses, but i can use the utils functions for that

Reply:
True, forgot about that.